front/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:18.15.0-alpine as front
```

Comment:
it's a very bare-bone Dockerfiles, there might be some ways to improve build speed (I'm not yet a Docker expert)